weforza/lib/extensions/date_extensions.dart

```diff
@@ -0,0 +1,16 @@
+extension SimpleToString on DateTime {
+  // Pad a number with zeroes so it becomes a two digit one.
+  String _padLeft(int number) => number.toString().padLeft(2,"0");
+
+  // Returns a simple date string.
+  // I.e. 2012-02-27 13:27:00
```

Comment:
- [ ] Mention that the output of `toStringSimple()` is compatible with `DataTime.parse()`.